spec.yaml

```diff
@@ -355,42 +355,47 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
-  /invite/accept:
+  /invite/forward:
     post:
-      summary: Accept an invite to a shared document
-      description: Foo
+      summary: Forward an invitation to start sharing
+      description: Forwards an invitation received by an out-of-band channel, email for example.
       parameters:
         - name: invite
           in: body
-          description: The JSON object to notify OCM that an invite has been accepted.
+          description: The JSON object representing an invitation to send to an end-user.
           required: true
           schema:
-            $ref: "#/definitions/AcceptInvite"
+            $ref: "#/definitions/Invite"
       responses:
         200:
-          description: Auth ok!
+          description: Invitation sent.
         400:
-          description: Invalid token?
+          description: |
+            Bad request due to invalid parameters, e.g. email is not valid.
           schema:
-            $ref: "#/definitions/Error" 
-  invite/forward:
+            $ref: "#/definitions/400"            
+  /invite/accept:
     post:
-      summary: Forward information from invite to the cloud provider.
-      description: Foo
+      summary: Accept an invititation to start sharing.
+      description: Accepts an invitation to the shared document so the provider can initiate the OCM share creation.
```

Comment:
I think this description needs to be updated, I'm not sure exactly how though. But isn't this an invitation to be able to share with someone ie in some sense decoupled from the sharing itself, like:
1. Invite A
2. Now one can share with A?

Reply:
Yes, you accept the invitation to create a share with the person. Invitation and share creation are both decoupled. We can remove the shared document part.